plugins/actions/_v2_config_template.py

```diff
@@ -34,7 +34,8 @@
 
 
 from ansible.plugins.action import ActionBase
-from ansible.utils.unicode import to_bytes, to_unicode
+#from ansible.utils.unicode import to_unicode # ansible.utils.unicode. is deprecated, use ansible.module_utils._text instead
```

Comment:
I don't think this is necessary

Reply:
should disappear since we are going to resync with upstream

Reply:
Oh, that ok! I was needed to deploy the cluster urgently, and I showed the solution to the problem, thank you.